.gitignore

```diff
@@ -1,2 +1,5 @@
 composer.lock
 vendor/
+
+# Covers JetBrains IDEs: IntelliJ, RubyMine, PhpStorm, AppCode, PyCharm, CLion, Android Studio
+.idea/
```

Comment:
sorry, it was for develop branch in my repo
and thanks for helpful advice )
fixed